cpp/tests/experimental/louvain_test.cu

```diff
@@ -77,8 +84,20 @@ class Tests_Louvain : public ::testing::TestWithParam<Louvain_Usecase> {
 
     auto graph_view = graph.view();
 
-    louvain(graph_view);
-#endif
+    // "FIXME": remove this check
+    //
+    // Disable louvain(experimental::graph_view_t,...)
+    // versions for GPU architectures < 700
+    // (cuco/static_map.cuh depends on features not supported on or before Pascal)
+    //
+    cudaDeviceProp device_prop;
+    CUDA_CHECK(cudaGetDeviceProperties(&device_prop, 0));
+
+    if (device_prop.major < 7) {
+      EXPECT_THROW(louvain(graph_view), cugraph::logic_error);
```

Comment:
Sorry if I missed it elsewhere, but are we filtering this test out for Pascal systems? (since we previously had it commented out of our cmake file)  If not, will this result in a CI failure on a Pascal system?

Reply:
No.  The test runs on all systems.  The `device_prop.major < 7` test identifies if the system is running Pascal or earlier.  If so then the call will throw an exception so the `EXPECT_THROW` will make this test pass.  If the system is running Volta or later then we take the else block that follows and we run the test without the `EXPECT_THROW`.

Reply:
Oh I see, thanks for the explanation. I was initially thinking we'd need to just skip the Pascal run (by upgrading to googletest 1.10.0 and calling `GTEST_SKIP()`), but this is better because it tests both code paths (supported platforms run louvain, unsupported throw an exception) properly.